index.html

```diff
@@ -129,7 +129,61 @@ <h2 class="ui divider horizontal">Ils l'utilisent déjà</h2>
         'contract'
       ],
       searchFullText: true,
-      source: data
+      source: data,
+      templates: {
+        message: function(type, message) {
+          html = '<p class="searchError">Aucune API ne correspond à votre recherche</p>'
```

Comment:
You don't have a variable declaration (no `var`). So the function scope is broken, and you leak that `html` variable in the global scope.
In this specific case, you don't actually need the intermediate variable. Just `return` the string concatenation.